### DISCUSSION TITLE
Can't Sign with personal_sign in MetaMask

### DISCUSSION THREAD

**Discussion by HubeauChou** [WalletConnect/walletconnect-monorepo#3133] [Developer Support]
I connected the wallet successfully, and when I went to request a signature, I couldn't effect any events.

**Comment by HubeauChou** [comment]
method == "personal_sign" {
            return AnyCodable(["0x4d7920656d61696c206973206a6f686e40646f652e636f6d202d2031363533333933373535313531", account])
        }

**Comment by finessevanes** [comment]
Could you provide the following:
1) Version of the package you are using
2) Reproducible code via a codeBox, error logs, and/or reproduction steps
3) Which wallet (platform and version) you used to get to the issue
4) Do you get the issues on https://react-web3wallet.vercel.app/

**Reply by HubeauChou** [reply]
Of course, I use the main branch. and connect MetaMask, it is available on https://react-web3wallet.vercel.app/

**Reply by HubeauChou** [reply]
An error was reported in the iOS code
